core/src/main/java/org/hisp/dhis/android/core/relationship/RelationshipStoreImpl.java

```diff
@@ -60,12 +60,7 @@
 
     private static final String QUERY_BY_UID = "SELECT " + FIELDS + " FROM " +
```

Comment:
the test download_tracked_entity_instance_enrollments_and_events_with_relationships is failing after merging this branch with development changes and I think this change is the problem because now only realize filter by trackedEntityInstanceA field but in the test, the Uid filter for this query is trackedEntityInstanceB in relation